src/Workspaces/Core/Portable/AddImports/AbstractAddImportsService.cs

```diff
@@ -79,12 +81,18 @@ public SyntaxNode GetImportContainer(SyntaxNode root, SyntaxNode contextLocation
         {
             contextLocation = contextLocation ?? root;
             GetContainers(root, contextLocation,
-                out var externContainer, out var usingContainer, out var aliasContainer);
+                out var externContainer, out var usingContainer, out var staticUsingContainer, out var aliasContainer);
 
             switch (import)
             {
                 case TExternSyntax e: return externContainer;
-                case TUsingOrAliasSyntax u: return IsAlias(u) ? aliasContainer : usingContainer;
+                case TUsingOrAliasSyntax u:
+                    if (IsAlias(u))
+                    {
+                        return aliasContainer;
+                    }
+
+                    return IsStaticUsing(u) ? staticUsingContainer : usingContainer;
```

Comment:
the mix of `if/ternary` is offputting to me :)  just pick one or the other.

Reply:
I have never found nested ternaries to be readable. =(